frontend/vue.config.js

```diff
@@ -78,28 +112,31 @@ module.exports = {
 };
 
 function formatThirdPartyLicenses(jsLicenses) {
-	// Remove the HTML character encoding caused by Handlebars
-	let licenses = rustLicenses.map((rustLicense) => ({
-		licenseName: htmlDecode(rustLicense.licenseName),
-		licenseText: trimBlankLines(htmlDecode(rustLicense.licenseText)),
-		packages: rustLicense.packages.map((package) => ({
-			name: htmlDecode(package.name),
-			version: htmlDecode(package.version),
-			author: htmlDecode(package.author).replace(/\[(.*), \]/, "$1"),
-			repository: htmlDecode(package.repository),
-		})),
-	}));
-
-	// De-duplicate any licenses with the same text by merging their lists of packages
-	licenses.forEach((license, licenseIndex) => {
-		licenses.slice(0, licenseIndex).forEach((comparisonLicense) => {
-			if (license.licenseText === comparisonLicense.licenseText) {
-				license.packages.push(...comparisonLicense.packages);
-				comparisonLicense.packages = [];
-				// After emptying the packages, the redundant license with no packages will be removed in the next step's `filter()`
-			}
+	let licenses = [];
+	if (rustLicenses) {
```

Comment:
`if (!rustLicenses) return;` would help avoid a scope indentation, which is helpful for something like this which is reasonably long.

Reply:
Previously, the code would output the javascript licenses, but add a notice that rust licenses are missing. With your proposed change, it would skip all license information, including the warning.

I tried to keep the logic intact.